libs/transloco/src/lib/transloco.service.ts

```diff
@@ -136,7 +145,7 @@ export class TranslocoService implements OnDestroy {
     /**
      * When we have a failure, we want to define the next language that succeeded as the active
      */
-    this.events$.pipe(takeUntilDestroyed()).subscribe((e) => {
```

Comment:
Please remove the import as it's the only usage in this file

Reply:
Still pipes the `loadTranslation` observable.

Reply:
Oh right 🤦  haha